drodrpg/DRODLib/Character.cpp

```diff
@@ -1807,6 +1807,9 @@ void CCharacter::Process(
 	CDbRoom& room = *(pGame->pRoom);
 	CSwordsman& player = *pGame->pPlayer;
 
+	// Some commands should not be executed when previewing a room remotely
```

Comment:
Let's update the comment to state
`// Some commands should not have side effects when previewing a room remotely`